types/adobe__pdfservices-node-sdk/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for adobe__pdfservices-node-sdk 2.2
+// Project: https://opensource.adobe.com/pdfservices-node-sdk-samples/apidocs/latest/index.html
+// Definitions by: Thom Kiesewetter <https://github.com/DefinitelyTyped>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Library is Generated from the jsdoc in the npm package.
+
+import * as lib from './pdfservices-sdk';
+export = lib;
```

Comment:
```suggestion
import * as PDFServicesSDK from './pdfservices-sdk';
export = PDFServicesSDK;
```
as literally exported symbol name, optional, aligns with module.